docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,555 @@
+
+## Get Started
+
+To make your own singing robot, you need the following hardware:
+
+- A computer with a webcam, speakers, and the [Go Client SDK](https://pkg.go.dev/go.viam.com/rdk) installed.
+  Tess used a Macbook, but you can use any PC with a Viam-compatible operating system that meets the above requirements.
+
+While following this tutorial, you'll complete the following steps to train and utilize a machine learning model on a webcam:
+
+1. [**Train your ML Model with pictures of toys:**](#train-your-ml-model-with-pictures-of-toys)
+
+- [Configure the camera on your laptop to capture data](#configure-your-webcam-to-capture-data) with the Data Management service and use it to capture images.
+   After that, filter your image data with [tags](#tag-data) and use it to train a [machine learning model](/services/ml/) on [the Viam app](https://app.viam.com).
+
+2. [**Use your ML Model to sing songs to your kids:**](#use-your-ml-model-to-sing-songs-to-your-kids)
+
+- [Configure your webcam to act as a shape classifier](#configure-your-webcam-to-act-as-a-shape-classifier) with Viam's [ML Model](/services/ml/) and [Vision](/services/vision/) Services, [record bedtime songs](#record-bedtime-songs), and [program the robot](#program-your-bedtime-songs-bot) to play a song when it recognizes a shape.
+
+## Train your ML Model with pictures of toys
+
+### Configure your webcam to capture data
+
+In the [Viam app](https://app.viam.com), create a new robot and follow the steps on your new robot’s **Setup** tab.
+
+Navigate to your robot's page on the app and click on the [**Config** tab](/manage/configuration/).
+
+First, add your personal computer's webcam to your robot as a [camera](/components/camera/) by creating a new component with **type** `camera` and **model** `webcam`:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+![Creation of a `webcam` camera in the Viam app config builder. The user is selecting the video_path configuration attribute from the drop-down menu](../../tutorials/bedtime-songs-bot/video-path-ui.png)
+
+You do not have to edit the attributes of your camera at this point.
+Optionally, select a fixed filepath for the camera from the automated options in the **video path** drop-down menu.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+``` json {class="line-numbers linkable-line-numbers"}
+"components": [
+  {
+    "model": "webcam",
+    "attributes": {},
+    "depends_on": [],
+    "name": "cam",
+    "type": "camera"
+  }
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Name your camera `"cam"`.
+If you use a different name, adapt the code in the later steps of this tutorial to use the name you give your camera.
+
+To view your webcam's image stream, navigate to the **Control** tab of your robot's page on [the Viam app](https://app.viam.com).
+Click on the drop-down menu labeled **camera** and toggle the feed on.
+Click on **Export Screenshot** to capture an image, as shown below:
```

Comment:
Why do they need to do this? (I think they don't?)

Reply:
Addressed w/ following thread